Library/Homebrew/cmd/help.rb

```diff
@@ -32,14 +32,34 @@
 # NOTE The reason the string is at the top is so 25 lines is easy to measure!
 
 module Homebrew
-  def help
-    puts HOMEBREW_HELP
-  end
+  def help(cmd = nil, empty_argv = false)
```

Comment:
I'd personally prefer `options={}` with an `[:empty_argv]` just for readability; boolean arguments are often hard to parse from their calling side otherwise.